WeakAurasOptions/ActionOptions.lua

```diff
@@ -150,6 +150,15 @@ function OptionsPrivate.GetActionOptions(data)
         disabled = function() return not data.actions.start.do_message end,
         hidden = function() return data.actions.start.message_type ~= "WHISPER" end
       },
+      start_message_tts_voice = {
+        type = "select",
+        width = WeakAuras.doubleWidth,
+        name = L["Voice"],
+        order = 3.2,
+        disabled = function() return not data.actions.start.do_message end,
+        hidden = function() return (not WeakAuras.IsRetail()) or data.actions.start.message_type ~= "TTS" end,
```

Comment:
An aura imported from retail to classic will have the option hidden but not disable, it will trigger an Lua error because `C_VoiceChat.SpeakText` will be nil

Reply:
@mrbuds that should be fixed now, yeah?

Reply:
Sorry, should've replied. The commit should cover that though I haven't had a chance to get on Classic to check. Would welcome anyone else doing that, otherwise I should be able to Monday Afternoon.

Reply:
I tested the build on bcc, looks good everything is working.